lib/phoenix/pubsub/pg2.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Phoenix.PubSub.PG2 do
+  use Supervisor
+
+  @moduledoc """
+  The Supervisor for the `:pg2` `Phoenix.PubSub` adapter
+
+  To use PG2 as your PubSub adapter, simply add it to your application's
+  supervision tree:
+
+      children = [
+        ...
+        worker(...),
+        supervisor(Phoenix.PubSub.PG2, [[name: MyApp.PubSub]]),
+      ]
+
+  ## Options
+
+    * `name` - The required name to register the PubSub processes, ie: `MyApp.PubSub`
+
+  """
+
+  def start_link(opts) do
+    supervisor_name = Module.concat(__MODULE__, Keyword.fetch!(opts, :name))
```

Comment:
I think we should use the same convention we use for local here. I.e.:

    Module.concat(Keyword.fetch!(opts, :name), Supervisor)